### PR TITLE
first

### PR DESCRIPTION
**Pull request by Artem18084**
- [DEMO LINK](https://Artem18084.github.io/layout_antisnake/)
- [TEST REPORT LINK](https://Artem18084.github.io/layout_antisnake/report/html_report/)

### CODE REVIEW

**Comment by Artem18084** [comment]
- [DEMO LINK](https://Artem18084.github.io/layout_antisnake/)
- [TEST REPORT LINK](https://Artem18084.github.io/layout_antisnake/report/html_report/)

File: src/styles/snake/antisnake.scss [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  }
+  &__cell--4 {
+    background-color: rgb(40%, 0%, 0%);
+  }
+  &__cell--5 {
+    background-color: rgb(20%, 0%, 0%);
+  }
+  &__cell--6 {
+    background-color: rgb(0%, 0%, 0%);
+  }
```
Comment (Olexandr6) [inline src/styles/snake/antisnake.scss:33]: 9-st point of the CHECKLIST 👉 [checklist.md](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md)

File: src/styles/snake/antisnake.scss [hunk 2]
```diff
@@ -0,0 +1,42 @@
+.antisnake {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+
+  &__cell {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 100px;
```
Comment (Olexandr6) [inline src/styles/snake/antisnake.scss:10]: It is good practice to set: font-family, font-size in html tag.

File: src/styles/snake/antisnake.scss [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+
+  &__cell {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 100px;
+    color: #fff;
```
Comment (Olexandr6) [inline src/styles/snake/antisnake.scss:11]: You should create a variable for this color.

File: src/styles/snake/antisnake.scss [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    height: 300px;
```
Comment (Olexandr6) [inline src/styles/snake/antisnake.scss:12]: You should create a variable for the cell size.

File: src/styles/snake/antisnake.scss (hunk 1 above)
Comment (Olexandr6) [reply src/styles/snake/antisnake.scss:33]: You can rewrite it by using @for or @each cycles, as well.

**Review by Olexandr6 — CHANGES_REQUESTED** [review]
You should fix the task. Also, you have some bug with cell width. The cell width always should be more than 300px.👇
image

File: src/styles/snake/antisnake.scss (hunk 4 above)
Comment (Olexandr6) [inline src/styles/snake/antisnake.scss:12]: You should create a variable for сell size.
```suggestion
    height: $cell-size;

```

**Review by Olexandr6 — CHANGES_REQUESTED** [review]

**Comment by Artem18084** [comment]
- [DEMO LINK](https://Artem18084.github.io/layout_antisnake/)
- [TEST REPORT LINK](https://Artem18084.github.io/layout_antisnake/report/html_report/)

File: src/styles/snake/antisnake.scss [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+  @for $index from 1 through 6 {
+    &__cell--#{$index} {
+      background-color: rgb(256 - 51 * ($index - 1) , 0, 0);
+      
+    }
+
+  }
+
+}
```
Comment (VinnieJ-2k20) [inline src/styles/snake/antisnake.scss:29]:
```suggestion
  @for $index from 1 through 6 {
    &__cell--#{$index} {
      background-color: rgb(256 - 51 * ($index - 1) , 0, 0);
    }
  }
}

```

**Review by VinnieJ-2k20 — APPROVED** [review]
Good job 👍

**Pull request closed without merging**